mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/presenter/ReadQrPresenter.java

```diff
@@ -7,20 +7,25 @@
 import javax.inject.Inject;
 
 /**
- * Created by naman on 7/9/17.
+ * ReadQr Presenter
+ * @author naman
+ * @since 8/7/17
  */
 
 public class ReadQrPresenter implements QrContract.ReadQrPresenter {
 
     private final UseCaseHandler mUsecaseHandler;
     private QrContract.ReadQrView mReadQrView;
 
-
+    
     @Inject
     public ReadQrPresenter(UseCaseHandler useCaseHandler) {
```

Comment:
Document this as well.

Reply:
This is just the default constructor does not need any explanation

Reply:
You can simply put `Initializes ReadQrPresenter Class`